omero/developers/Python.txt

```diff
@@ -39,11 +39,13 @@ All the code examples below can be found at
 :sourcedir:`examples/Training/python`.
 
 
-Start by downloading the first example below: 
-:source:`examples/Training/python/Connect_To_OMERO.py` and edit the
-USERNAME and PASSWORD variables according to your log-in.
+If you want to run the examples, you will need to download and
+configure them to connect to your own server. E.g. ``HOST = "localhost"``
+You can edit ``HOST, PORT, USERNAME and PASSWORD`` in
+the ``Parse_OMERO_Properties.py`` file and these values will be imported
+into the other scripts.
```

Comment:
I can see it either way. For power developers, I would argue `ice.config` files should be presented as the default way in order to create connections as opposed to hard-coding credentials in scripts. But for training purposes, telling people to modify the top-level script is certainly fine.